includes/html/api_functions.inc.php

```diff
@@ -1600,6 +1619,7 @@ function list_oxidized(Illuminate\Http\Request $request)
              })
              ->whereNotIn('type', Config::get('oxidized.ignore_types', []))
              ->whereNotIn('os', Config::get('oxidized.ignore_os', []))
+             ->whereNotIn('devices.device_id', $ignoredDeviceIds)
```

Comment:
Once you have device ids in the config setting, you can save two sql queries and delete all the above code with:
```suggestion
             ->whereDoesntHave('group', fn ($query) => $query->whereIn('id', Config::get('oxidized.ignore_device_groups')))
```